charts/sampling/templates/_otelcol_processor_transform.river.txt

```diff
@@ -0,0 +1,69 @@
+{{- define "deployment.processor.transform.add_resource_attributes" -}}
+otelcol.processor.transform "add_resource_attributes_as_metric_attributes" {
+  // https://grafana.com/docs/agent/latest/flow/reference/components/otelcol.processor.transform/
+  error_mode = "ignore"
+
+  metric_statements {
+    context    = "datapoint"
+    statements = [
+      "set(attributes[\"deployment.environment\"], resource.attributes[\"deployment.environment\"])",
+      "set(attributes[\"service.version\"], resource.attributes[\"service.version\"])",
+    ]
+  }
+
+  output {
+    metrics = [otelcol.processor.batch.default.input]
+  }
+}
+
+{{ end }}
+
+{{- define "deployment.processor.transform.add_k8s_attributes" -}}
+otelcol.processor.transform "add_k8s_attributes" {
```

Comment:
Ditto, is this for incoming logs&traces or for logs&traces produced internally?
I'm not sure to catch the role for a "sampling box".